src/Goetas/Twital/Attribute/AttrAttribute.php

```diff
@@ -13,9 +14,9 @@
  */
 class AttrAttribute implements Attribute
 {
-    public static function getVarname(\DOMNode $node)
+    public static function getVarname(\DOMElement $node)
     {
-        return "__a9" . strtr(md5($node->ownerDocument->saveXML($node)).spl_object_hash($node), "-","_");
+        return "__a9" . strtr($node->getAttributeNS(Twital::NS, '__internal-id__').spl_object_hash($node), "-","_");
```

Comment:
Wasn't the whole point of this to replace spl_object_hash with some unique key? Or maybe I just don't understand how it works now.